source/Body.h

```diff
@@ -35,8 +35,10 @@ class Body {
 public:
 	// Constructors.
 	Body() = default;
-	Body(const Sprite *sprite, Point position, Point velocity = Point(), Angle facing = Angle(), double zoom = 1.);
-	Body(const Body &sprite, Point position, Point velocity = Point(), Angle facing = Angle(), double zoom = 1.);
+	Body(const Sprite *sprite, Point position, Point velocity = Point(), Angle facing = Angle(),
+		double zoom = 1., double alpha = 1.);
+	Body(const Body &sprite, Point position, Point velocity = Point(), Angle facing = Angle(),
+		double zoom = 1., double alpha = 1.);
```

Comment:
Why are you changing both of these constructuors? You only seem to be using the first one.